udata/core/user/commands.py

```diff
@@ -46,6 +47,20 @@ def create():
 
 
 @m.command
+def activate():
+    '''Activate an existing user (validate their email confirmation)'''
+    email = prompt('Email')
```

Comment:
:+1: for consistency.
Proposal: they all takes an optional first argument email (login in fact) that will be prompted if not provided